src/pages/dataTableUtilities/columns.js

```diff
@@ -7,13 +7,42 @@ import { tableStrings } from '../../localization';
 
 const PAGE_COLUMN_WIDTHS = {
   customerInvoice: [2, 4, 2, 2, 1],
+  customerInvoices: [1.5, 2.5, 2, 3, 1],
 };
 
 const PAGE_COLUMNS = {
   customerInvoice: ['itemCode', 'itemName', 'availableQuantity', 'totalQuantity', 'remove'],
+  customerInvoices: ['serialNumber', 'otherPartyName', 'status', 'comment', 'delete'],
 };
 
 const COLUMNS = () => ({
+  serialNumber: {
+    key: 'serialNumber',
+    title: tableStrings.invoice_number,
+    sortable: true,
+  },
+  otherPartyName: {
+    key: 'otherPartyName',
+    title: tableStrings.customer,
+    sortable: true,
+  },
+  status: {
+    key: 'status',
+    title: tableStrings.status,
+    sortable: true,
+  },
+  entryDate: {
+    // TODO: unbork it
```

Comment:
tbh I hate `//TODO`'s a lot - they never get do'd and just sit there forever if there's no issue made (and even with an issue hehe)

Reply:
That's fair. This one is a bit different cause I know that your branches potentially have the code to do this, so me doing it is a bit derp. Could make an issue "CIs page needs entryDate column" if you like?

Reply:
I think there will be a couple things - navigation & status column(?) maybe just a new issue to revisit the page?

Reply:
https://github.com/openmsupply/mobile/issues/1167